p/scripts/main.js

```diff
@@ -1093,6 +1093,36 @@ function init_stream(stream) {
 			return true;
 		}
 
+		el = ev.target.closest('.item.share a.dropdown-toggle');
+		if (el) {
+			const itemId = el.closest('.flux').id;
+			const templateId = 'share_article_template';
+			const id = itemId;
```

Comment:
Check with https://github.com/FreshRSS/Extensions/issues/263#issuecomment-2546636861

Reply:
`main.js` is JavaScript. But it is a PHP issue there.

Reply:
https://github.com/FreshRSS/FreshRSS/pull/7113